toolkit/mozapps/handling/ContentDispatchChooser.jsm

```diff
@@ -227,18 +227,32 @@ let ContentDispatchChooserTelemetry = {
   },
 };
 
+XPCOMUtils.defineLazyModuleGetters(this, {
+  MigrationUtils: "resource:///modules/MigrationUtils.jsm",
+});
+
 const replaySchemeMap = {
-  library: 'https://replay.io/view'
+  library: 'https://replay.io/view',
+  migrate: (uri, principal, browsingContext) => {
+    const win = browsingContext.topFrameElement.getTabBrowser().owerGlobal;
+    MigrationUtils.showMigrationWizard(win, [
+      MigrationUtils.MIGRATION_ENTRYPOINT_UNKNOWN,
+    ]);
+  }
```

Comment:
Want to include a trailing comma so we don't have to modify this line as we add more entries?

Reply:
yup!